etc/adminhtml/system.xml

```diff
@@ -34,6 +34,11 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     <comment>If enabled, the import will wait for any running indexers to complete before starting the import process. With this setting off, the import will fail/cancel if any indexers are running when it starts</comment>
                 </field>
+                <field id="clear_attribute_code" translate="label" type="select" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Clearly this config option is related to attributes, so why have you put it in the 'general' group, instead of 'attributes'?